code/player/TTTPlayer.cs

```diff
@@ -74,7 +53,7 @@ public override void Respawn()
             EnableShadowInFirstPerson = true;
 
             Role = new NoneRole();
-            Credits = 0;
+            Credits = 400;
```

Comment:
just another magic number maybe we want to throw as a server convar.

Reply:
Just for testing, as I don't plan giving every player Credits in the start if we are done.